.changeset/flat-mugs-try.md

```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/dashboard": patch
```

Comment:
question: shouldn't there be more packages here. 

@medusaja/product and @medusajs/types?